client/transactions/list/test/index.tsx

```diff
@@ -415,16 +426,28 @@ describe( 'Transactions list', () => {
 			} );
 		} );
 
-		afterEach( () => {
-			jest.resetAllMocks();
-		} );
+		test( 'should request confirmation when the download button is clicked', () => {
```

Comment:
May use a better name here to indicate that the confirmation popup will only display if the to-be-downloaded number of transactions is bigger than the threshold 1000?

Reply:
I changed the names of this test (and the new test to cover the case when merchants choose not to proceed): https://github.com/Automattic/woocommerce-payments/pull/3477/commits/aa7647f2a055b23da9341ba3413e8d4b059ae13f